src/ui-kit/InnerTable/InnerTable.tsx

```diff
@@ -0,0 +1,30 @@
+import { JSX } from '@redneckz/uni-jsx';
+import { useTariffsInnerTableData } from '../../hooks/useTariffsInnerTableData';
+import type { UniBlockProps } from '../../types';
+import { InnerTableBody } from './InnerTableBody';
+import { TableInnerButton } from './InnerTableButton';
+import { InnerTableHeader } from './InnerTableHeader';
+import { TableFetchData, TariffsTableInnerContent } from './InnerTableProps';
+
+export interface InnerTableProps extends TariffsTableInnerContent, UniBlockProps {}
+
+export const InnerTable = JSX<InnerTableProps>(({ context, dataUrl }) => {
+  const { tableData }: TableFetchData = useTariffsInnerTableData(context.useAsyncData, dataUrl);
+
+  const [activeTabIndex, setActiveTabIndex] = context.useState(0);
+  const currencies = tableData?.map((item) => item.currency ?? '');
+  const items = tableData?.[activeTabIndex]?.items ?? [];
```

Comment:
Не один логический блок, нужны отступы, либо связанные вычисления поставить рядом, а не связанные отделить
```suggestion
  const [activeTabIndex, setActiveTabIndex] = context.useState(0);
  const items = tableData?.[activeTabIndex]?.items ?? [];
  
  const currencies = tableData?.map((item) => item.currency ?? '');
```